src/components/sdk/DeviceSelection/CameraSelection/index.tsx

```diff
@@ -14,25 +13,18 @@ interface Props extends BaseSdkProps {
   notFoundMsg?: string;
   /** The label that will be shown for camera selection, it defaults to "Camera source". */
   label?: string;
-  /** A boolean that determines whether or not to include additional sample video devices, such as "None", "Blue", "SMTP Color Bars". Defaults to true. This will be deprecated in the next major version. */
-  appendSampleDevices?: boolean;
 }
 
 export const CameraSelection: React.FC<Props> = ({
   notFoundMsg = 'No camera devices found',
   label = 'Camera source',
-  appendSampleDevices = true,
   ...rest
 }) => {
   const meetingManager = useMeetingManager();
-  const videoInputConfig: DeviceConfig = {
-    additionalDevices: appendSampleDevices,
+  const { devices, selectedDevice } = useVideoInputs();
+  const selectVideoInput = async (deviceId: string): Promise<void> => {
+    await meetingManager.selectVideoInputDevice(deviceId);
```

Comment:
Instead of meetingManager.selectVideoInputDevice can we just use the `useSelectVideoInputDevice` hook? Trying to unify the way we select these devices with a common pattern.

Reply:
I see that we do it in SpeakerSelection hence wondering if we can do it or if out of scope for now and needs lot of changes?

Reply:
I will try to unify them later. I think there should not be too many changes